### ISSUE TITLE
[20210725] Weekly AI ArXiv 만담

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#18]
- AI News
  - ICCV notification out: accept letter 받으신 모든 분들 축하드립니다!
  - IBM Watson 의 현주소 
    - NYT: https://www.nytimes.com/2021/07/16/technology/what-happened-ibm-watson.html
    - 국내: https://news.naver.com/main/read.naver?mode=LSD&mid=sec&sid1=105&oid=023&aid=0003627854
  - AlphaFold protein DB 공개: https://alphafold.ebi.ac.uk/
  - Tensor RT 8: https://docs.nvidia.com/deeplearning/tensorrt/release-notes/tensorrt-8.html
  
- AI Arxiv
  - [All That's 'Human' Is Not Gold: Evaluating Human Evaluation of Generated Text](https://arxiv.org/abs/2107.00061)
    - Outstanding paper in ACL2021
    - GPT3같은 고성능 NLG 모델의 성능 평가할 때는 평가자들에게 미리 훈육을 해야한다. 아니면 결과를 믿기 어려움
    - 예) 평가자들이 NLG모델이 문법 같은 것들이 틀릴 것이라는 가정을 하고 있지만 사실 이거 잘못됐음.그래서 많이 틀림
    - 결론은 NLG와 사람이 쓴 글 예시를 몇개 보여주는 example-based 훈련 하고 진행하면 그래도 정확도가 통계적으로 좋아지더라
    - 주의할점. GPT3-175B의 경우 훈육 안하면 정확도 50%(랜덤 추측), 훈육해도 57% ㄷㄷㄷ
    
  - [Beyond Goldfish Memory: Long-Term Open-Domain Conversation](https://arxiv.org/abs/2107.07567)
    - Blender Bot 2.0의 주요논문 1 - 금붕어 기억력 탈출
    - Multi session chat 데이터: 크라우드 소싱 활용해서 situated 시간 점프 채팅
    - Token 길이를 얼마나 두느냐 (이전 대화 활용). Summary를 활용하느냐 (Gold vs. Predicted)
![image](https://user-images.githubusercontent.com/11782739/126858867-b7fd1728-b629-4182-8932-b62bbe2a27c4.png)
![image](https://user-images.githubusercontent.com/11782739/126858880-bb8632be-d326-49fb-a99f-50925bc8e977.png)
    
  - [Internet-Augmented Dialogue Generation](https://arxiv.org/abs/2107.07566)
    - Blender Bot 2.0의 주요논문 2 - 인터넷을 활용한 대화 생성.
    - Retrieval-augmented generation 을 넘은 Internet-augmented generation
    - 최신성, hallucination 두가지 문제를 해결 가능
    - Wizard of the Internet dataset
    - 실제 인터넷 search를 하면 document level information이 나오기 때문에 이를 잘 summary 하는 것이 숙제
![image](https://user-images.githubusercontent.com/11782739/126883266-04fafb50-d87b-456d-b3b0-eac086f63a65.png)
![image](https://user-images.githubusercontent.com/11782739/126883282-899ca43c-9e4a-4438-bbc1-e450a49b83c0.png)
![image](https://user-images.githubusercontent.com/11782739/126883288-18e6f9e8-cc96-4c3f-9269-452c128275d4.png)
![image](https://user-images.githubusercontent.com/11782739/126883348-585bb0ae-6593-47ab-b7af-58149b622afd.png)

  - [TumorCP: A Simple but Effective Object-Level Data Augmentation for Tumor Segmentation](https://arxiv.org/abs/2107.09843v1)
    - copy-paste 기반의 간단한 medical data augmentation 
    - source-target pair 고르고 object 뜯은 다음 spatial, gamma, blurring을 확률 적으로 transform후 image-level augmentation
    - IntraCP and InterCP (환자내, 환자간)
    - kidney, tumor 데이터 적용, 데이터 숫자 때문인지 Tumor에선 꽤 효과가 좋아 보임.
    - https://github.com/YaoZhang93/TumorCP (아직은 껍데기만)
![image](https://user-images.githubusercontent.com/11782739/126883474-4abea9bb-582c-4bad-b924-5558f70539f5.png)
![image](https://user-images.githubusercontent.com/11782739/126883518-f854b6b1-a381-468d-a40f-74f33a5d6114.png)

  - [An overview of mixing augmentation methods and augmentation strategies](https://arxiv.org/abs/2107.09887v1)
    - Image classification 백본 훈련에서 Data augmentation 총정리
    - CIFAR-10, 100, ImageNet 에서 최근 나온 DA 기법까지 총망라
    - the pixel-wise approaches (e.g. Mixup) work better with noise (corrupted images or incorrect labels) while the patchwise ones (e.g. CutMix ) are better suited to the task of partial occlusion or weakly supervised object localization problem.
    - 결국은 잘 섞어서 사용하세요 라는.. 전체적으로 cutmix가 간단하면서도 매우 효과적임을 확인 가능

  - [Ready for Emerging Threats to Recommender Systems? A Graph Convolution-based Generative Shilling Attack](https://arxiv.org/abs/2107.10457v1)
    - 추천시스템의 robustness 를 attack 하는 연구
    - Shilling attack : 대량의 가짜 유저 프로파일을 injection 
    - 예전엔 너무 단순해서 효과가 부실하거나.. 효과는 있으나 cost가 많이 들어 실효성 없었다고
    - GCN이용 co-related items correlation과 가짜 점수 스무드를 하는 GAN을 이용하 real rating distribution 을 배움. 
![image](https://user-images.githubusercontent.com/11782739/126887104-9bd8e23b-4404-4d6a-b7bd-6e135821c148.png)

  - [Triplet is All You Need with Random Mappings for Unsupervised Visual Representation Learning](https://arxiv.org/abs/2107.10419v1)
    - Triplet loss + CE loss 로 negative sample 하나만 활용하는 SSL
    - 미니배치 사이즈 클 필요 없음. SimCLR, SimSiam 과 얹어서 활용가능
    - similarity function 에 random matrix를 활용해서 M을 decompose 한 공간상에서도 학습이되도록하는 효과
    - 리포팅 된 실험 데이터 사이즈가 작은 게 아쉬운데 ImageNet-1k 수준에서 체크해보고 싶은..
![image](https://user-images.githubusercontent.com/11782739/126887372-9582687a-56bf-4ba9-8b29-48684763cc50.png)
![image](https://user-images.githubusercontent.com/11782739/126887392-3f45d5f6-2ab1-4742-8e9a-0da3a485f8d9.png)

**Comment by sihyun-yu** [comment]
https://ai.stanford.edu/blog/prototransformer/
코딩 수업에 대한 피드백을 주는 문제를 few-shot classification으로 보고 학습시킨 뒤 실제 deploy도 했다고 합니다. 관심 있으신 분들 홈페이지에 나와있는 디테일 읽어보시면 좋을 것 같습니딘.

**Comment by laray5959** [comment]
### Paper

- [Sequence-to-Sequence Piano Transcription with Transformers](https://arxiv.org/abs/2107.09142)
![seq2seqpiaon](https://github.com/laray5959/PaperList/blob/master/piano_transcription.PNG?raw=true)
    - Sample : 
    - Conference :
    - Problem : 
    - Contribution :  
    - Method 

- [StarGANv2-VC: A Diverse, Unsupervised, Non-parallel Framework for Natural-Sounding Voice Conversion](https://arxiv.org/abs/2107.10394)
![starganv2vc](https://github.com/laray5959/PaperList/blob/master/stargenv2_vc.PNG?raw=true)
    - Sample : 
    - Conference :
    - Problem : 
    - Contribution :  
    - Method 

- [Digital Einstein Experience: Fast Text-to-Speech for Conversational AI](https://arxiv.org/abs/2107.10658)
![einstein](https://github.com/laray5959/PaperList/blob/master/einstein.PNG?raw=true)
    - Homepage : https://einstein.digitalhumans.com/ 
    - Blog : https://www.aflorithmic.ai/post/creating-einsteins-voice
    - Conference : INTERSPEECH 2021
    - Problem : 
    - Contribution :  
        - Digital Einstein chatbot -real-time으로 einstein 목소리와 표정을 생성 
    - Method 
        - FastSpeech2를 이용하였고 log-scaled mel-spectrogram을 예측
        - ParallelWaveGAN 사용

**Comment by Kyung-Min** [comment]
Paper
- User-specific Adaptive Fine-tuning for Cross-domain Recommendations
  - 추천시스템의 cross domain transfer를 위한 finetuning 방법
  - 타겟 데이터셋 크기가 작을 때, 파라미터가 매우 큰 모델을 통째로 finetuning하면 overfitting이 발생하기 쉬움
  - 유저별로 서로 다른 레이어를 tuning해줌으로써, finetuning cost를 줄이고, performance도 향상시킬 수 있었다라고 주장
  - input sequence가 들어왔을 때 어떤 레이어를 tuning해줄지 결정하는 policy networks 존재
  - hard (gumbel softmax) / soft (gating mechanism) / RL (REINFORCE style)
 
News
- https://www.wsj.com/video/series/inside-tiktoks-highly-secretive-algorithm/investigation-how-tiktok-algorithm-figures-out-your-deepest-desires/6C0C2040-FF25-4827-8528-2BD6612E3796?mc_cid=3453acec1b&mc_eid=b3e8464478
  - Tiktok’s algorithm analysis
    - Toppop -> niche contents that makes you stay longer on the platform (in few hours), 
    - View time optimization (watching for longer time doesn’t mean you really like it but it is your vulnerability)

**Comment by veritas9872** [comment]
**Neuroprosthesis for Decoding Speech in a Paralyzed Person with Anarthria**

https://www.nejm.org/doi/full/10.1056/NEJMoa2027540

AI가 드디어 독심술에 성공했습니다! ㄷㄷㄷㄷㄷㄷ
New England Journal of Medicine (NEJM)에서 마비가 찾아온 사람의 뇌신호로부터 단어 및 문장을 예측하는데 성공했습니다.
아직 paywall 뒤에 있지만 조만간 큰 뉴스가 될 것으로 예상됩니다.
몇 년 후에 뇌내 AI에 의해 의사소통하는 기술의 시조가 될지도 모릅니다 ㅎㅎㅎ

**Comment by kwonminki** [comment]
논문 소개는 아니지만... 크롬을 사용하시는 분들께 공유하면 좋을 듯 해서 올려봅니다.

https://chrome.google.com/webstore/detail/arxive/hkoblclipggkhhbllgefhnbjdcajmelh?hl=ko

chrome extension으로 'Arxive' 라고 하나 만들어서 올려봤습니다. 크롬 웹스토어에서 직접 검색하셔도 나옵니다.

정말 간단하게 arxiv 사이트를 통해 논문을 받을 때 파일 제목을 논문제목 + 저자 + 연도로 저장할 수 있게 만들었습니다.
arxiv 사이트에 접속하면 원래 논문을 다운받던 pdf 밑에 Direct download가 페이지에 하나 생기고 해당 링크를 클릭하시면 됩니다.
설정에서 저자, 연도 유무 선택이 가능하고, 바로 다운로드 폴더에 받을지 다른이름으로 저장할지도 설정 가능합니다.
필요하신 분 있으시면 쓰시면 좋을 듯 합니다. (나중에 시간나면 다른 사이트들도 추가해볼까 합니다)
